deps/restore_git_submodules.props

```diff
@@ -2,11 +2,9 @@
   <PropertyGroup>
     <RestoreGitSubmodulesImported>true</RestoreGitSubmodulesImported>
   </PropertyGroup>
-  <Target Name="RestoreGitSubmodules" BeforeTargets="PrepareForBuild">
```

Comment:
in fact, this part was bad before too. _People just need to learn to do submodule updates before they build._ I'm sorry. :)